Fixing Subtract Function

abelRoland opened the pull request:
This closes #5

abelRoland committed:
Fixing Subtract Function
This closes #5

Sayed94h commented on scripts/handlers/subtract.js:
```diff
@@ -83,10 +83,26 @@ function subtractHandler() {
+    if (Object.is(inputConfirmed, NaN)) { alert('Be sure you are entering a number! Try again. '); }
+    else {
+      // subtract the last result by the user's number and reassign lastResult
+      const result = subtract(inputConfirmed, lastResult);
```
Hi Abel, 
there is a shared variable called 'lastResult'. Here in this line, the variable 'lastResult' should be reassigned instead creating a new const variable 'result'. And also the lastResult should be subtracted from the user's input(just reverse the arguments inside the function).

Thank you

Sayed94h reviewed (changes requested):
Hi Abel, there is a small issue in the code. I added comment for to make it clear.
Thank you. Greetings

Sayed94h commented on scripts/handlers/subtract.js:
```diff
@@ -12,7 +12,7 @@ function subtract(x, y) {
+  const result = x - y;
 
   if (typeof result !== 'number') { throw new TypeError('result'); }
   return result;
```
Below this line there should be a "}" to close the subtract(x, y) function

abelRoland replied:
ok. Thanks

abelRoland committed:
Update subtract.js

I believe that this fixes the issue. Please, see if everything is alright now @Sayed94h .

Thanks

abelRoland commented:
I believe that now is ready to be merged.
